src/client/datascience/jupyter/jupyterNotebookProvider.ts

```diff
@@ -30,31 +30,20 @@ export class JupyterNotebookProvider implements IJupyterNotebookProvider {
         return server?.getConnectionInfo();
     }
 
-    public async createNotebook(options: GetNotebookOptions): Promise<INotebook> {
+    public async createNotebook(options: NotebookCreationOptions): Promise<INotebook> {
         // Make sure we have a server
         const server = await this.serverProvider.getOrCreateServer({
-            getOnly: options.getOnly,
+            getOnly: false,
```

Comment:
Do we use getOnly anymore? Can we just remove it from the options type?

Reply:
For Jupyter server it's required, to do with prewaring.